QuantLib/ql/time/schedule.hpp

```diff
@@ -90,11 +103,12 @@ namespace QuantLib {
         //@}
       private:
         bool fullInterface_;
```

Comment:
it is still used in the fullInterface() inspector (other than I described
my plans on the Mailing list), to keep that backward compatible in case any
client code should rely on this inspector (at leat in the library,
test-suite and examples only the schedule class itself seems to use it).

On 28 April 2015 at 09:43, Francois Botha <notifications@github.com> wrote:

> In QuantLib/ql/time/schedule.hpp
> <https://github.com/lballabio/quantlib/pull/230#discussion_r29220231>:
>
> > @@ -90,11 +103,12 @@ namespace QuantLib {
> >          //@}
> >        private:
> >          bool fullInterface_;
>
> The fullInterface_ variable is no longer used. Perhaps remove it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lballabio/quantlib/pull/230/files#r29220231>.
>